packages/components/src/components/FoldOut/style.tsx

```diff
@@ -28,6 +32,26 @@ const StyledFoldOut = styled.div`
             `;
         }
 
+        if (props.excerptHeight && !props.isOpen) {
+            return css`
+                overflow: hidden;
+                position: relative;
+
+                &:after {
+                    content: '';
+                    position: absolute;
+                    left: 0;
+                    right: 0;
+                    top: 0;
+                    bottom: 0;
+                    background: linear-gradient(
+                        transparent 50%,
+                        ${props.backgroundColor ? props.backgroundColor : '#FFFFFF'} 100%
```

Comment:
You can shorten this to: 
```suggestion
                        ${props.backgroundColor || '#FFFFFF'} 100%
```